draft-ietf-quic-recovery.md

```diff
@@ -802,7 +772,7 @@ Pseudocode for SetLossDetectionTimer follows:
         timeout = 2 * kInitialRtt
       else:
         timeout = 2 * smoothed_rtt
-      timeout = max(timeout, kMinTLPTimeout)
+      timeout = max(timeout, granularity)
```

Comment:
```suggestion
      timeout = max(timeout, kGranularity)
```

Reply:
If you want to use granularity, then you need to define it above.  Otherwise, use kGranularity as suggested.